CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Added new checker option `mypy-options` in `static-type-checker` to let users override default mypy command-line arguments
 - Added documentation for overriding messages
 - Improved `check_contracts` error messages by ensuring a consistent format and phrasing
+- Added `autoformat-options` configuration option to let users specify command-line arguments to the Black formatting tool
```

Comment:
This should be moved into the "unreleased" section (I recently made a new release for 2.10.0 and 2.10.1)